requirements.txt

```
@@ -1,14 +1,23 @@
-asgiref==3.4.1
-click==8.0.1
-fastapi==0.66.0
-h11==0.12.0
-httptools==0.2.0
-pydantic==1.8.2
-python-dotenv==0.18.0
-pyyaml==5.4.1
-starlette==0.14.2
+#
+# These requirements were autogenerated by pipenv
+# To regenerate from the project's Pipfile, run:
+#
+#    pipenv lock --requirements
+#
+
+-i https://pypi.org/simple
+aiohttp==3.7.4.post0
+async-timeout==3.0.1; python_full_version >= '3.5.3'
+atoma==0.0.17
+attrs==21.2.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
+chardet==4.0.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
+defusedxml==0.7.1; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
+fastapi==0.67.0
+idna==3.2; python_version >= '3.5'
+multidict==5.1.0; python_version >= '3.6'
+pydantic==1.8.2; python_full_version >= '3.6.1'
+python-dateutil==2.8.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+six==1.16.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+starlette==0.14.2; python_version >= '3.6'
 typing-extensions==3.10.0.0
-uvicorn==0.14.0
-uvloop==0.15.2
-watchgod==0.7
-websockets==9.1
+yarl==1.6.3; python_version >= '3.6'
```

Could you please add `atoma` and `twitter`?
```suggestion
yarl==1.6.3
twitter==1.19.3
atoma==1.16.0
python_version >= '3.6'
```

---

just did